docs/configuration.asciidoc

```diff
@@ -551,7 +551,18 @@ A capture location stack trace is never generated for uncaught exceptions.
 * *Default:* `true`
 * *Env:* `ELASTIC_APM_CAPTURE_SPAN_STACK_TRACES`
 
-Set this option to `false` to disable capture of stack traces for measured spans during instrumentation.
+Set this option to `false` to disable capture of stack traces.
+
+When set to `true` the agent will capture stack traces for spans whose duration falls within the <<span-frames-min-duration, `spanFramesMinDuration`>> threshold.
+
+[[span-frames-min-duration]]
+==== `spanFramesMinDuration`
+
+* *Type:* String
+* *Default:* `10ms`
+* *Env:* `ELASTIC_APM_CAPTURE_SPAN_STACK_TRACES_THRESHOLD`
+
+A time value to control whether the agent will report stack traces for measured spans. If a span's duration reaches this threshold, the agent will report stack traces.
```

Comment:
Would be helpful to mention interaction with `captureSpanStackTraces` setting... for those that link directly to this anchor and don't notice that `captureSpanStackTraces` is defined just above.

Reply:
Sounds like an improvement to me.  Done. ✅